system-test/bigtable.ts

```diff
@@ -1111,8 +1130,152 @@ describe('Bigtable', () => {
       assert.strictEqual(rows.length, 0);
     });
   });
+
+  describe('backups', () => {
+    const CLUSTER = INSTANCE.cluster(CLUSTER_ID);
+    let BACKUP: Backup;
+
+    // For these tests, two backups are needed. The backups are labeled for what
+    // they are intended to originate/interact from/with, but this is just for
+    // testing and the naming convention used here does not actually influence
+    // the real functionality - it is just a way to keep things organized!
+    const backupIdFromCluster = generateId('backup');
+    const backupNameFromCluster = `${CLUSTER.name}/backups/${backupIdFromCluster}`;
```

Comment:
Does it pay to use `replaceProjectId` or [`replaceProjectIdToken`](https://github.com/stephenplusplus/nodejs-bigtable/blob/spp--backups/src/index.ts#L15) here and a few lines below for backupNameFromTable?